src/index.ts

```diff
@@ -1,10 +1,10 @@
-import * as dotenv from "dotenv";
+// Before running the app, loads all the environment variables from a .env files
+// @see https://github.com/motdotla/dotenv/tree/v6.1.0#how-do-i-use-dotenv-with-import
+import "./env";
+
 import newApp from "./app";
 import { log } from "./utils/logger";
 
-// Before running the app, loads all the environment variables from a .env files
-dotenv.config();
```

Comment:
this is better than the side effect in `env.ts` as it makes clear what is going on